### ISSUE TITLE
BEP causeing high CPU load?

### ISSUE THREAD

**Issue by timlinux** [kartoza/least-cost-electrification-benin#48]
I'm not sure if it is the BEP but I tried to enable schools and my CPU fan started running hard. I looked at the performance monitor and it looks like  on CPU is going full bore.

![image](https://user-images.githubusercontent.com/178003/92331306-8e6f9400-f06d-11ea-907b-45bc5709377a.png)

After I removed the tab where I was trying to show schools it went back down.

![image](https://user-images.githubusercontent.com/178003/92331337-c2e35000-f06d-11ea-9cb8-bf27cfed5c81.png)

Can you try to independently verify this @meomancer ?

**Comment by meomancer** [comment]
Yes, i tried it too, the problem is from cluster layer with "Stand-alone - Photovoltaic", the Grid cluster is fine
i'm not sure the source for this, maybe the number of vertex? (there are 62054 features in map)
